pkg/repository/local_git_test.go

```diff
@@ -20,9 +20,39 @@ import (
 	"testing"
 	"time"
 
+	"github.com/okteto/okteto/pkg/ignore"
+	"github.com/spf13/afero"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
 )
 
+type mockLocalGit struct {
```

Comment:
Even if we already have a `mockLocalExec`, I wasn't be able to use it as I had to mock 2 calls to `RunCommand` in the same test, and it wasn't allowing it. So, instead of modifying all the tests, I created another implementation using testify mocks for simplicity